react-validated-input-component/src/password.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+
+export default class Password extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: '' };
+    this.handleChange = this.handleChange.bind(this);
+    this.iconClass = "fas fa-times red";
+    this.message = "A password is required";
+  };
+  handleChange(event) {
+    this.setState({ value: event.target.value });
+  };
+  render() {
+    if (this.state.value.length > 0 && this.state.value.length < 9) {
+      this.message = "Your password is too short"
+      this.iconClass = "fas fa-times red";
+    }
+    if (this.state.value.length > 8) {
```

Comment:
Here you just need an `else` if the else is hit you already know it is greater than 8 because you checked if it was less than 9 above